astroplan/tests/test_core.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord)
+import astropy.units as u
+from astropy.time import Time
+from astropy.tests.helper import remote_data
```

Comment:
Yep, using `remote_data` like this is the way to go, @bmorris3.  

@cdeil - see astropy/astropy#3890 for a PR where I've added the relevant module.  I didn't realize that testing stuff wasn't anywhere in the docs...